### PR TITLE
poc metadata refactoring (GDEV-1565) 

### PR DESCRIPTION
**Pull request by KerstenBreuer**
Technical specification for the next metadata epic. The crayfish epic will be adapted in a separate PR.

### CODE REVIEW

File: 25-pacific_lamprey/technical_specification.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Scope:
+The aim of this study is to create a proof of concept implementation of the strategy
+outlined in
+https://docs.ghga-dev.de/main/architecture_concepts/ac002_metadata_lifecycle.html.
+
+
+## Additional Implementation Details:
+
+- A implementation that can be run by data stewards locally,
```
Comment (lkuchenb) [inline 25-pacific_lamprey/technical_specification.md:14]:
```suggestion
- An implementation that can be run by data stewards locally,
```

File: 25-pacific_lamprey/technical_specification.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - basic in implementation of the submission store
+    - Validation of metadata upon submission against the linkML schema
+    - Essential transformations and generation of associated artifacts, including:
+        - Fully embedded datasets
+        - Custom embeddings
+        - Non-restricted (public) metadata
+        - Summary statistics
+        - Mongodb-based search index
+    - REST APIs for querying the above artifacts
+- not includes:
```
Comment (lkuchenb) [inline 25-pacific_lamprey/technical_specification.md:26]:
```suggestion
- does not include:
```

File: 25-pacific_lamprey/technical_specification.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- includes:
+    - basic in implementation of the submission store
+    - Validation of metadata upon submission against the linkML schema
+    - Essential transformations and generation of associated artifacts, including:
+        - Fully embedded datasets
+        - Custom embeddings
+        - Non-restricted (public) metadata
+        - Summary statistics
+        - Mongodb-based search index
+    - REST APIs for querying the above artifacts
```
Comment (lkuchenb) [inline 25-pacific_lamprey/technical_specification.md:25]: Does this include code that serves this API or only API design? How is this to be seen in the context of a tool that is to be "run by data stewards locally"?

**Review by lkuchenb — CHANGES_REQUESTED** [review]
Thanks, see comments above!

File: 25-pacific_lamprey/technical_specification.md (hunk 3 above)
Comment (Cito) [reply 25-pacific_lamprey/technical_specification.md:25]: As discussed, if it includes code that serves the API, then add side-loading the generated artifiacts into the database as separate point before this one.

**Review by Cito — DISMISSED** [review]

**Review by Cito — APPROVED** [review]

**Review by lkuchenb — APPROVED** [review]

**Pull request merged**